TACC/Core-Portal
# task/FP-181 Download Attachments in Ticket history

## Overview: ##
Currently users can create or reply to tickets and provide file attachments (screenshots etc).  Users should be able to either view or download these attachments.
## Related Jira tickets: ##

* [FP-181](https://jira.tacc.utexas.edu/browse/FP-181)

## Summary of Changes: ##
Add new function on backend (views.py, urls.py, rtUtil.py) to get attachment files and edit TicketModal.js to display on frontend.
## Testing Steps: ##
1. Go to My Dashboard
2. Open or create any ticket with an attachments
3. See if you download the attachments from the ticket.

## UI Photos:
![Screen Shot 2021-10-01 at 6 56 26 PM](https://user-images.githubusercontent.com/34716439/135889882-ca1f34b7-0cc9-46eb-a62e-d4ca2cfb94fc.png)

## Notes: ##
